app/src/components/blocks/lgDownloadComplete/LgDownloadComplete.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { PatientDetails } from '../../../types/generic/patientDetails';
+import { Button, Card } from 'nhsuk-react-components';
+import { LG_RECORD_STAGE } from '../../../pages/lloydGeorgeRecordPage/LloydGeorgeRecordPage';
+
+export type Props = {
+    patientDetails: PatientDetails;
+    setStage: Dispatch<SetStateAction<LG_RECORD_STAGE>>;
+    stage?: LG_RECORD_STAGE; // For unit tests only
+};
+
+function LgDownloadComplete({ patientDetails, stage, setStage }: Props) {
+    return (
+        <>
+            <Card style={{ maxWidth: '620px' }} data-testid={`${stage}`}>
```

Comment:
Should we not add a css class for the maxWidth attribute rather than inline styling?